index.js

```diff
@@ -17,8 +17,14 @@
  \*───────────────────────────────────────────────────────────────────────────*/
 'use strict';
 
-var amdBuilder = require('makara-writer-amd').amdBuilder;
-
+var path = require('path');
+var iferr = require('iferr');
+var spundle = require('spundle');
+var moduleBuilder = function (appRoot, m, cb) {
+    spundle(path.resolve(appRoot, 'locales'), m[2], m[1], iferr(cb, function (out) {
```

Comment:
Thanks. I added a sanity check in the main returned function below for the locale parts